OpenSourceOrg/dotOrg
# Update/wide template changes width all elements

#### Changes proposed in this Pull Request

* Makes sure the new wide template max-width is set for all elements throughout. Some elements were getting a smaller width due to specific css styles overriding the content width. This PR addresses that

#### Testing instructions

* Open the Endorsements page (or create a new page + set it with the new No header - Wide template)
* Make sure there is some text + headings + forms in the page
* CHeck that they all become the full width of the page and aren't inhering the width of the rest of the site

Mentions #5002
